packages/open-truss/src/configuration/engine-v1.tsx

```diff
@@ -27,21 +31,29 @@ export function engineV1(
   config: WorkflowV1,
   data: YamlType,
```

Comment:
Not new to this PR but it looks like this `data` isn't used and the name is overridden on line 31 in `(config, data) => {` so perhaps we should remove it up here?

Reply:
Oh good idea. We can always add that back in whenever we're ready to use it. 👍 

```suggestion
```